src/PackageVersions/Installer.php

```diff
@@ -48,6 +48,12 @@ final class Installer implements PluginInterface, EventSubscriberInterface
 %s
 {
     public const ROOT_PACKAGE_NAME = '%s';
+    /**
+     * Array of all available composer packages.
```

Comment:
Can you also add the type of this array? It is `array<string, string>`

Reply:
sure